libraries/ESP8266WebServer/src/ESP8266WebServer.cpp

```diff
@@ -36,11 +36,9 @@
 #define DEBUG_OUTPUT Serial
 #endif
 
-//const char * AUTHORIZATION_HEADER = "Authorization";
 static const char AUTHORIZATION_HEADER[] PROGMEM = "Authorization";
 static const char qop_auth[] PROGMEM = "qop=auth";
 static const char WWW_Authenticate[] PROGMEM = "WWW-Authenticate";
-static const char colon[] PROGMEM = ":";
```

Comment:
Thanks for containing your OCD here.

Reply:
:) there's another one on L94 that could spare 2 more flash bytes